rusk-wallet/src/bin/io/prompt.rs

```diff
@@ -199,6 +199,15 @@ pub(crate) fn ask_confirm() -> anyhow::Result<bool> {
     Ok(a.as_bool().expect("answer to be a bool"))
 }
 
+/// Asks the user for confirmation before deleting cache
+pub(crate) fn ask_confirm_erase_cache() -> anyhow::Result<bool> {
+    let q = requestty::Question::confirm("confirm")
+        .message("Corrupt cache detected, delete the cache? (Alternatively specify --profile to change the location)")
```

Comment:
What happens if the cache is deleted? Are there any downsides that the user needs to be informed of? I vaguely remember that @herr-seppia wanted to add some sort of disclaimer here

Reply:
If the cache's deleted then user will have to re run the wallet to create new cache. All this info is logged along with the disclamair Seppia mentioned regarding the --profile flag

Reply:
> All this info is logged along with the disclamair Seppia mentioned regarding the --profile flag

I don't see a disclaimer, what are you referring to?

Reply:
> I don't see a disclaimer, what are you referring to?
"Alternatively specify --profile to change the location"